app/views/admin/split_details/edit.html.erb

```diff
@@ -1,27 +1,21 @@
-<% content_for :page_title, @split_detail.split.name %>
+<% content_for :body_layout_body_color, 'white' %>
+<% content_for :page_title, "Split Details: #{@split_detail.split.name}" %>
 
 <div class="TakeoverText">
   <h1 class="TakeoverText-title">Context and outcomes</h1>
 </div>
 
-<div class="InfoCard">
-  <%= simple_form_for @split_detail, url: admin_split_details_path, method: :put do |f| %>
-    <%= f.error :base %>
-    <%= f.input :owner, label: "Squad Name" %>
-    <%= f.input :platform, collection: Split.platforms.keys %>
-    <%= f.input :location, label: "Location", hint: "Example: Sign Up" %>
+<%= simple_form_for @split_detail, url: admin_split_details_path, method: :put do |f| %>
+  <%= f.error :base %>
+  <%= f.input :owner, label: "Squad Name" %>
```

Comment:
we talked about not calling this "Squad Name," right? we were going to simplify it to "Owner" or something?